python/tvm/exec/pick_best.py

```diff
@@ -0,0 +1,38 @@
+# pylint: disable=invalid-name
+"""Pick best log entries from a large file and store them to a small file"""
```

Comment:
From the file name, I cannot tell what does this script do, consider have a better file name. e.g.

```bash
python -m tvm.exec.autotvm_log_editor --pick-best xxx
```